src/utils/cart/cart.js

```diff
@@ -0,0 +1,22 @@
+const isItemInCart = (cart, id) => {
+  return cart.find(cartInItem => cartInItem._id === id);
+}
+
+const getPriceDetails = (cart) => {
```

Comment:
This is a really nice way of delegating the code, will make this change in my code too.